pmd-java/src/main/resources/category/java/codestyle.xml

```diff
@@ -1366,14 +1366,27 @@ public class ClassInDefaultPackage {
         <priority>3</priority>
         <properties>
             <property name="version" value="2.0"/>
+            <property name="acceptableLength" type="Integer" value="4"
```

Comment:
You need to add a `min` & `max` attribute for it to work

Reply:
Also, since we plan on extending the rule to other bases, naming it "acceptableDecimalLength" would make it future-proof since we'd probably add e.g. "acceptableHexLength"